apps/dashboard/app/javascript/packs/batch_connect.js

```diff
@@ -423,16 +424,16 @@ function clamp(currentValue, previous, next) {
     return undefined;
 
   // you've set the boundary, so when you go to the next value - keep it at the next's boundary
-  } else if(previous && previous['max'] && currentValue == previous['max']) {
-    return next['max'];
-  } else if(previous && previous['min'] && currentValue == previous['min']) {
+  } else if(currentValue === previous['min']) {
```

Comment:
`previous['max']` can be 0 which evaluates to fasley - so just stop checking for it's existance because javascript deals well with comparing `undefined` or `null` against actual objects.

This also makes sure we check the minimum before the maximum so things get clamped to the minimums.